### ISSUE TITLE
Request for a `/members/{memberid}/journals` route

### ISSUE THREAD

**Issue by sckott** [CrossRef/rest-api-doc#117]
It would be great to be able to get journals for a publisher. We can get prefixes for a publisher, but not journals. Or can we?  Seems one could do it with `filter` on `/journals` route, but it's not supported unless using `/journals/{issn}/works`

**Comment by Bubblbu** [comment]
Any chance this will be usable any time soon?

Currently, I've only come across the quite inelegant way of accessing `/members/{memberid}/works` and retrieving all unique ISSNs from the list of works.

Is there a better way of getting to the list of ISSNs for a publisher?

**Comment by ppolischuk** [comment]
@Bubblbu Would the ISSN facet help in your case? For example, http://api.crossref.org/members/2984/works?facet=issn:*

**Comment by Bubblbu** [comment]
yes, awesome! I somehow hadn't considered facet counts... but yes! this does the job. thanks :)

**Comment by Bubblbu** [comment]
I've come all the way round to encounter this problem again and am now realizing that the ISSN facet is limited to 100 results.

I was wondering if there is a way to get _all_ journals of a particular publisher.

As @sckott said a few years ago, using a `member` filer on `/journals` is not supported leaving a few inefficient workarounds
